[14-18-55] [ENTRY]: [Nathan]
[14-18-55] [SYSTEM_IO]: System data pared in IOUtil's static block
[14-18-55] [SYSTEM_IO]: Suggestions pared in IOUtil's static block
[14-18-55] [SYSTEM_IO]: Exit conditions pared in IOUtil's static block
[14-18-55] [SYSTEM_IO]: DebugHashes pared in IOUtil's static block
[14-18-56] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.updateOldJson(UserUtil.java:759)
From line: 759
Trace: java.lang.NullPointerException
	at cyder.utilities.UserUtil.updateOldJson(UserUtil.java:759)
	at cyder.utilities.IOUtil.fixUsers(IOUtil.java:386)
	at cyder.genesis.Cyder.main(Cyder.java:93)
[14-18-56] [CORRUPTION]: userdir saved to: [Resulting Popup]
Unfortunately a user's data file was corrupted and had to be deleted. The following files still exists and are associated with the user at the following path:<br/>Cyder/dynamic/54ee27f4-6ce8-3636-8825-43bd794c7fd3<br/>Files:<br/>Default
[14-18-57] [LOGIN]: [AUTOCYPHER ATTEMPT]
[14-18-57] [JAVA ARGS]: [LOCATION] [La casa de Nathan]
[14-18-57] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.<init>(CyderFrame.java:119)
	at cyder.handlers.internal.PopupHandler.inform(PopupHandler.java:46)
	at cyder.handlers.internal.PopupHandler.inform(PopupHandler.java:26)
	at cyder.handlers.internal.PopupHandler.inform(PopupHandler.java:22)
	at cyder.utilities.UserUtil.userJsonDeleted(UserUtil.java:921)
	at cyder.utilities.UserUtil.fixUser(UserUtil.java:227)
	at cyder.ui.ConsoleFrame.setUUID(ConsoleFrame.java:1693)
	at cyder.utilities.UserUtil.checkPassword(UserUtil.java:633)
	at cyder.handlers.internal.LoginHandler.recognize(LoginHandler.java:393)
	at cyder.handlers.internal.LoginHandler.autoCypher(LoginHandler.java:468)
	at cyder.handlers.internal.LoginHandler.beginLogin(LoginHandler.java:365)
	at cyder.genesis.Cyder.main(Cyder.java:112)
[14-18-57] [CORRUPTION]: userdir saved to: [Resulting Popup]
Unfortunately a user's data file was corrupted and had to be deleted. The following files still exists and are associated with the user at the following path:<br/>Cyder/dynamic/54ee27f4-6ce8-3636-8825-43bd794c7fd3<br/>Files:<br/>Default
[14-18-58] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.getUserData(UserUtil.java:456)
From line: 456
Trace: java.lang.IllegalArgumentException: userdata.json does not exist
	at cyder.utilities.UserUtil.getUserData(UserUtil.java:456)
	at cyder.ui.ConsoleFrame.setUUID(ConsoleFrame.java:1695)
	at cyder.utilities.UserUtil.checkPassword(UserUtil.java:633)
	at cyder.handlers.internal.LoginHandler.recognize(LoginHandler.java:393)
	at cyder.handlers.internal.LoginHandler.autoCypher(LoginHandler.java:468)
	at cyder.handlers.internal.LoginHandler.beginLogin(LoginHandler.java:365)
	at cyder.genesis.Cyder.main(Cyder.java:112)
[14-18-58] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.setUserData(UserUtil.java:123)
From line: 123
Trace: java.lang.IllegalArgumentException: File does not exist
	at cyder.utilities.UserUtil.setUserData(UserUtil.java:123)
	at cyder.handlers.internal.LoginHandler.recognize(LoginHandler.java:398)
	at cyder.handlers.internal.LoginHandler.autoCypher(LoginHandler.java:468)
	at cyder.handlers.internal.LoginHandler.beginLogin(LoginHandler.java:365)
	at cyder.genesis.Cyder.main(Cyder.java:112)
[14-18-58] [LOGIN]: [AUTOCYPHER FAIL]
[14-18-58] [WIDGET OPENED]: LOGIN
[14-19-46] [ACTION]: Component name = [cyder.ui.DragLabel], bounds = [(2, 2, 1164, 28)], hash = [1997702454], parentFrame = [Userdata Corruption], associated text = [No getText() method found], tooltip text = [No getTooltipText() method found], title = [No getTitle() method found]
[14-19-46] [ACTION]: CyderFrame disposed with fastclose: false, CyderFrame: Component name = [cyder.ui.CyderFrame], bounds = [(696, 620, 1168, 152)], hash = [586358252], parentFrame = [Object itself is the top level frame], associated text = [No getText() method found], tooltip text = [No getTooltipText() method found], title = [Userdata Corruption]
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.getUserData(UserUtil.java:456)
From line: 456
Trace: java.lang.IllegalArgumentException: userdata.json does not exist
	at cyder.utilities.UserUtil.getUserData(UserUtil.java:456)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1113)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-19-46] [EXCEPTION]: 
Exception origin: cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
From line: 394
Trace: java.lang.IllegalArgumentException: Provided file does not exist
	at cyder.utilities.UserUtil.extractUser(UserUtil.java:394)
	at cyder.ui.CyderFrame.repaint(CyderFrame.java:1642)
	at cyder.ui.CyderFrame.refreshBackground(CyderFrame.java:1498)
	at cyder.ui.CyderFrame.setBounds(CyderFrame.java:1380)
	at java.desktop/java.awt.Component.move(Component.java:2111)
	at java.desktop/java.awt.Component.setLocation(Component.java:2093)
	at java.desktop/java.awt.Window.setLocation(Window.java:924)
	at cyder.ui.CyderFrame.lambda$dispose$4(CyderFrame.java:1125)
	at java.base/java.lang.Thread.run(Thread.java:844)
[14-20-07] [EOL]: Log completed, Cyder was force closed by an external entity: exit code: -200 [External Stop], exceptions thrown: 21